rosdep/base.yaml

```diff
@@ -3811,6 +3811,7 @@ libjsoncpp:
   rhel: [jsoncpp]
   ubuntu:
     '*': [libjsoncpp1]
```

Comment:
The wildcard rule should be used for the latest value and expected potential future values. (aka jammy and kinetic are 25 and the best chance of a future version being the same are that value.) 

So please update libjsoncpp25 to be the wildcard and fill in all the older supported versions, which looks like libjsoncpp1 for bionic and focal. Older platforms can be pruned as they are EOL.

Reply:
I left all "current" Ubuntu LTS distros (so xenial/trusty/bionic/focal as it seems that xenial/trusty are still in extended support mode) and updated wildcard for jammy/future versions.